app/models/export.rb

```diff
@@ -24,8 +24,16 @@ def total_storage
     account.media_attachments.sum(:file_file_size)
   end
 
+  def total_statuses
+    account.statuses_count
+  end
+
   def total_follows
-    account.following.count
+    account.following_count
+  end
+
+  def total_followers
+    account.followers_count
   end
```

Comment:
Maybe use `account.followers.size` etc. (instead of `account.followers.count`)? From RoR's documentation, RoR will use the cache appropriately.

Reply:
It's because the account was not reloaded before checking. The #count method was not cached before this PR so it worked, but followers_count is an attribute so it had to be reloaded.

Reply:
Wouldn't `.size` (which properly uses the cache according to RoR documentation) handle this correctly, using the cache while avoiding the need to reload? (It's a real question, I don't know the answer)

Reply:
The reason I'm switching to followers_count from followers.count (size is an alias of that btw, there is no difference) is to avoid a super expensive query.

Reply:
Sorry, but this is not true. See https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache: “With this declaration, Rails will keep the cache value up to date, and then return that value in response to the size method.”

Reply:
We're not using Rails counter cache anyway.

Reply:
Then what are we using?

Reply:
#7644

Reply:
Sorry but `following_count` and `followers_count`, which we are talking about here, are still using `counter_cache`.

Reply:
Ah, my bad.